### PR TITLE
feat: made in brazil collection

### PR DESCRIPTION
**Pull request by caarlos0**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

Brazilians have created a lot of cool projects, and I think this curation would be nice.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by kiliczsh** [comment]
Ping me when this PR is merged please @caarlos0

File: collections/made-in-brazil/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - dracula/dracula-theme
+  - elixir-lang/elixir
+  - ellisonleao/magictools
+  - felipefialho/frontend-challenges
+  - go-task/task
+  - heartcombo/devise
+  - htop-dev/htop/
+  - lua/lua
+  - RocketChat/Rocket.Chat
+  - tsuru/tsuru
```
Comment (MikeMcQuaid) [inline collections/made-in-brazil/index.md:14]:
```suggestion
  - tsuru/tsuru
  - goreleaser/goreleaser
```

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by caarlos0** [comment]
(just fixed the flag to be a square as it seems to be required)

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @caarlos0!

**Comment by MikeMcQuaid** [comment]
https://github.com/collections/made-in-brazil 🎉

**Comment by caarlos0** [comment]
Thanks for the review/merge @MikeMcQuaid 🎉

**Pull request merged**